web/src/features/hfiCalculator/components/DailyHFICell.tsx

```diff
@@ -0,0 +1,17 @@
+import HFICell from 'components/HFICell'
+import CalculatedCell from 'features/hfiCalculator/components/CalculatedCell'
+import React from 'react'
+
+export interface DailyHFICellProps {
+  testid: string | undefined
+  value: string | undefined
+  error: boolean
+  className: string | undefined
+}
+
+export const DailyHFICell = (props: DailyHFICellProps) => {
```

Comment:
what's the justification for having a `HFICell` component and a `DailyHFICell` component (I'm just struggling to tell the difference between them?

Reply:
Daily View Table has this pattern with using a `CalculatedCell` for handling the case where there is an error. I just wrapped `CalculatedCell` and `HFICell` to abstract that away.